apophis/processes/reminders.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from core.readers import get_reminders
+import asyncio
+import logging
+
+
+async def output_reminder(context):
+    logging.info("Polling reminders..")
+    while True:
+        await asyncio.sleep(1)
```

Comment:
(pending review comment about time resolution) we don't need to poll every second. we should calculate the amount of time to the 1st second of the next minute and sleep for that long.

Reply:
So, could we not just poll every 61 seconds, rather than amount of time to the first second of the next minute?